src/components/AvatarPopup.js

```diff
@@ -0,0 +1,24 @@
+import Popup from "./Popup";
+
+class avatarPopup extends Popup {
```

Comment:
Class `avatarPopup` is not needed because you need to use `PopupWithForm` for it. It's a modal with a form

Reply:
`class avatarPopup `

classes are written in `UpperCase`